### PR TITLE
Add motionless

### PR DESCRIPTION
**Pull request by raybellwaves**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble be please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by raybellwaves** [comment]
@fmaussion would you like to be added as a maintainer?

**Comment by raybellwaves** [comment]
Not sure why it's saying `ModuleNotFoundError: No module named 'six'` here
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=244098&view=logs&j=10bda42a-d420-5c28-a43f-bc4e66405e5c&t=bef775d0-a1a2-55a3-9721-17147029baf0&l=570

File: recipes/motionless/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=2.7
+  run:
+    - python >=2.7
+    - setuptools
```
Comment (synapticarbors) [inline recipes/motionless/meta.yaml:24]: It doesn't look like `setuptools` is a run time dependency, so you can remove this here. You do need to add `six` as this is a requirement for the this version.

File: recipes/motionless/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: http://github.com/ryancox/motionless
+  summary: An easy way to generate Google Static Map URLs with Python.
+  license: Apache-2.0
+  license_file: LICENSE
```
Comment (synapticarbors) [inline recipes/motionless/meta.yaml:39]: Add comment:

```yaml
# License file manually packaged. See: https://github.com/ryancox/motionless/pull/46
license_file: LICENSE
...
```

**Comment by fmaussion** [comment]
Hi! I just wanted to add that I have now maintainer rights on PyPi and that motionless is now up to date there: https://pypi.org/project/motionless/

Is there anything else that needs to be done? Thanks!

**Comment by raybellwaves** [comment]
@conda-forge-admin please rerender

**Comment by raybellwaves** [comment]
@conda-forge-admin, please restart ci

**Comment by dopplershift** [comment]
I tried power-cycling to restart CI, but the underlying repo is gone, so no re-open. If this is still relevant, you'll need to submit a new PR.

**Comment by fmaussion** [comment]
@TimoRoth would you have time to carry this further? I think motionless is useful enough to be in conda-forge (or maybe there is an alternative I don't know about).

**Comment by TimoRoth** [comment]
I have a recipe for this on our oggm channel.
Should not be too hard to adapt, I'll have a look.

**Pull request closed without merging**